benchmarks/tests/test_views.py

```diff
@@ -44,7 +44,7 @@ def test_public_model(self):
 
     def test_non_public_model(self):
         resp = self.client.get("http://localhost:8000/model/2")
-        self.assertEqual(resp.status_code, 404)
+        self.assertEqual(resp.status_code, 200)
```

Comment:
this doesn't feel right: if a model is not public and not submitted to the competition, the url should not work.

I think what you're trying to do is to add another test that opens a model that is not public but submitted to the competition